### PR TITLE
Add OWNERS to nptests

### PR DESCRIPTION
**Pull request by mkoderer**
As discussed with gmarek we add an OWNERS file for nptests to make
it possible to merge stuff more quickly.

### CODE REVIEW

**Comment by wojtek-t** [comment]
@bowei  - since so far you were mostly maintaining this, I would like you to LGTM it before we approve (that you're fine with the following set of approvers).

**Comment by gonzolino** [comment]
@bowei Do you have any comments to this?

File: network/benchmarks/netperf/OWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+approvers:
+- gmarek
+- shyamjvs
+- bowei
+- mkoderer
+- gonzolino
```
Comment (bowei) [inline network/benchmarks/netperf/OWNERS]: I don't see any contributions from gonzolino in this repo?

**Comment by bowei** [comment]
Hi -- it is typically the case that contributors first are placed in the reviewers before becoming approvers. I would be happy to put you guys in the reviewers section for a while and then move to approvers.

**Comment by wojtek-t** [comment]
Given @bowei `s comment (which I agree with), I'm closing this PR - please open a new one if you want to add people as reviewers.

**Pull request closed without merging**